lib/fog/vsphere/models/compute/cdroms.rb

```diff
@@ -0,0 +1,43 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Cdroms < Fog::Collection
+        autoload :Cdrom, File.expand_path('../cdrom', __FILE__)
```

Comment:
Had another look at this, all other models do the autoloading this way. Where would you define that, @plribeiro3000 ?